Assets/Scripts/ScriptableObjects/TowerScriptableObject.cs

```diff
@@ -19,4 +19,9 @@ public TurretPrefabConstruction InstantiateConstructionTower(PlayerStateControll
         controller.PrepareTurret(spawnedConstructionTower.GetComponent<Interactable>());
         return spawnedConstructionTower.GetComponent<TurretPrefabConstruction>();
     }
+
+    public int getCost()
```

Comment:
```suggestion
    public int GetCost()
```
😋

Also, as the `cost` field is already public, should we either set it to private (with `[SerializeField]`), or just remove this method altogether? 🤔

Reply:
`cost` should be private, as it makes `SellTower()` read better IMO. Will push a commit adressing this asap